### PR TITLE
Like and comment

### PR DESCRIPTION
**Pull request by alineammann**
Has some bugs but like & comment is working.

### CODE REVIEW

**Review by jstnceronio — CHANGES_REQUESTED** [review]
Would move comment & comment-form to pages/post/comment

File: marketplace/src/app/pages/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -23,12 +31,20 @@ export class DashboardComponent implements OnInit {
...
     await this.router.navigateByUrl('/create', {
       state: { title: title }
     })
   }
+
+  getPosts() {
+    return this.dataService.getPosts();
+  }
+  updatePost() {
+    //return this.posts.filter(t => !t.doneDate).length / this.todoListService.todos.length * 100;
```
[the remark this answers is not in the document]
Comment (alineammann) [reply marketplace/src/app/pages/dashboard/dashboard.component.ts:46]: No, forgot to take it out

**Review by jstnceronio — APPROVED** [review]
I would've still created subfolders under post/comment but that's something for the backlog ...

**Comment by Akjo03** [comment]
closed #33

**Comment by Akjo03** [comment]
closes #33

**Pull request merged**